lib/diff.js

```diff
@@ -10,12 +10,14 @@ class GraphQLDiff {
      * @param {GraphQLObjectType|GraphQLScalarType|GraphQLEnumType|GraphQLNonNull|GraphQLList|GraphQLUnionType} otherType the GraphQL type instance which was compared to thisType
      * @param {string} diffType the specific kind of difference between thisType and otherType
      * @param {string} description
+     * @param {boolean} backwardsCompatible
```

Comment:
true if this diff is a non-breaking change when interpreted as thisType changing to otherType